xml/networking-configure_mtu.xml

```diff
@@ -167,18 +169,33 @@
             tags:
             - neutron.networks.vxlan</screen>
     <para>
-     This will set the physical interface managed by &product; that has
-     the GUEST network group tag assigned to it. This can be found in the
-     interfaces_set.yml file under the interface-models section.
+     This will set the physical interface managed by &product; that has the
+     GUEST network group tag assigned to it. This can be found in the
+     <filename>interfaces_set.yml</filename> file under the interface-models
+     section.
     </para>
    </step>
    <step>
     <para>
-     Next, remove the "dhcp-option-force=26,1400" line from
+     Edit <filename>neutron.conf.j2</filename> found in
+     <filename>~/openstack/my_cloud/config/neutron/</filename> to set
+     <literal>global_physnet_mtu</literal> to <literal>9000</literal> under
+     <literal>[DEFAULT]</literal>:
+    </para>
+<screen>[DEFAULT]
+...
+global_physnet_mtu = 9000</screen>
+    <para>
+     This allows Neutron to advertise the optimal MTU to instances (based upon
+     global_physnet_mtu minus the encapsulation size).
+    </para>
+   </step>
+   <step>
+    <para>
+     Remove the "dhcp-option-force=26,1400" line from
```

Comment:
Should this be marked up into literal also? I'd suggset so: <literal>dhcp-option-force=26,1400</literal>